elements/pfe-clipboard/pfe-clipboard.ts

```diff
@@ -104,6 +105,17 @@ export class PfeClipboard extends LitElement {
   @property({ type: String, reflect: true, attribute: 'copy-from' })
     copyFrom: string|'url'|'property' = 'url';
 
+  /**
+   * Specify if current site is behind [https]
+   */
+  @state() private _secure = window.isSecureContext;
+
+  /**
+   * Specify the copy type to be used depending on security and browser support
+   */
+  @property({ type: String || null }) copyType: 'navigator'|'queryCommand'| null = null;
```

Comment:
```suggestion
  @property({ type: String }) copyType: 'navigator' | 'queryCommand'  | null = null;
```

we don't need the `null` in the property declaration. `type` here is a shorthand for `converter`. see https://lit.dev/docs/components/properties/#property-options